test/rpc/validation.js

```diff
@@ -0,0 +1,59 @@
+const test = require('tape')
+const request = require('supertest')
+
+const { startRPC, closeRPC } = require('./helpers')
+const { middleware } = require('../../lib/rpc/validation')
+
+test('should comlete the request with non-params if the method does not require any params', t => {
```

Comment:
comlete -> complete 😄

Reply:
I find both of these test description with this "non-params" wording a bit hard too read, can we make it a bit more descriptive here what happens, since there is a lot of ground for confusion around around what is meant by "param" in this case.

Some first suggestion:

'should work with no request parameter "params" provided if the API endpoint has no parameter input'

But feel free to improve on that.

Reply:
a suggestion:
```suggestion
test('should work without `params` when it\'s optional', t => {
```

Reply:
Thank you!
I think it is better `should work without `params` when it\'s optional`.
and I have installed type checker ;)